package.json

```diff
@@ -122,13 +122,13 @@
         "@babel/preset-typescript": "^7.24.7",
         "@babel/template": "^7.24.7",
         "@babel/types": "^7.24.7",
-        "@box/blueprint-web": "^7.30.1",
-        "@box/blueprint-web-assets": "^4.21.0",
+        "@box/blueprint-web": "^8.1.0",
+        "@box/blueprint-web-assets": "^4.28.0",
         "@box/box-ai-content-answers": "^0.57.1",
         "@box/cldr-data": "^34.2.0",
         "@box/frontend": "^10.0.0",
         "@box/languages": "^1.0.0",
-        "@box/metadata-editor": "^0.65.0",
+        "@box/metadata-editor": "^0.71.0",
```

Comment:
do we need to update peer dependency list?

Reply:
Not necessarily as it takes latest from host app.
I'll do it either way and try to match versions we have currently in our app.